integration_tests/tests/dbt_project.py

```
@@ -260,16 +260,110 @@ def seed(self, data: List[dict], table_name: str):
+        logger.info(
+            "ClickHouse fix: rebuilding %s.%s with Nullable columns: %s",
+            schema,
+            table_name,
+            nullable_columns,
+        )
+
+        ch_query(f"DROP TABLE IF EXISTS {schema}.{tmp_name}")
+        ch_query(
+            f"CREATE TABLE {schema}.{tmp_name} "
+            f"ENGINE = MergeTree() ORDER BY tuple() "
+            f"AS SELECT {select_sql} FROM {schema}.{table_name}"
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)